hail/python/hail/methods/statgen.py

```diff
@@ -307,6 +308,9 @@ def linear_regression_rows(y, x, covariates, block_size=16, pass_through=()) ->
     -------
     :class:`.Table`
     """
+    if isinstance(Env.backend(), SparkBackend):
```

Comment:
shouldn't this be `if not`?

Reply:
Yep, duh. Thanks. That would have been not so good.